tests/wdio/secondaryNavigation-spec.js

```diff
@@ -4,55 +4,3 @@ Terra.describeViewports('secondary nav', ['tiny', 'huge'], () => {
     Terra.validates.element({ selector: '#root' });
   });
 });
-
-Terra.describeViewports('secondary scroll multiple files', ['huge'], () => {
```

Comment:
Was this functionality removed?

Reply:
This code now lives in terra-application in the secondary navigation layout. Not appropriate to test here anymore.